Packs/ApiModules/Scripts/JSONFeedApiModule/JSONFeedApiModule.py

```diff
@@ -333,6 +335,50 @@ def determine_indicator_type(indicator_type, auto_detect, value):
     return indicator_type
 
 
+def merge_indicators_service(first_indicator, second_indicator):
+    """
+    Gets two indicators with the same value and different services (tags) and merges them.
+    Args:
+        first_indicator: (dict) The first indicator.
+        second_indicator: (dict) The second indicator.
+    Returns:
+        A dict of the merged indicator.
+    """
+    first_indicator['rawJSON']['service'] += f", {second_indicator.get('rawJSON', {}).get('service', '')}"
+    return first_indicator
+
+
+def deduplicate_indicators(indicators):
+    """
+    Finds the indicators with the same value and different services and merge them.
+    Args:
+        indicators: (list) The all indicators fetched from the feed.
+    Returns:
+        A List of all unique indicators.
+    """
+    indicators_value_list = [indicator.get('value') for indicator in indicators]
+    indicators_value_set = set(indicators_value_list)
+
+    if len(indicators_value_set) < len(indicators_value_list):
+        unique_indicators = []
+        unique_indicators_indexes = {}
+
+        for indicator in indicators:
+            indicator_value = indicator.get('value')
+
+            if indicator_value in indicators_value_set:
+                unique_indicators_indexes[indicator_value] = len(unique_indicators)
+                unique_indicators.append(indicator)
+                indicators_value_set.remove(indicator_value)
+            else:
+                indicator_index = unique_indicators_indexes.get(indicator_value)
+                unique_indicators[indicator_index] = merge_indicators_service(unique_indicators[indicator_index], indicator)
```

Comment:
If indicator is not in indicators_value_set then indicator_index will be None and if it's also not in unique_indicators then line 375 file will fail